### ISSUE TITLE
Handling for arbitrary number of reference sections/probes

### ISSUE THREAD

**Issue by klapo** [klapo/pyfocs#29]
As we extend pyfocs to use dtscalibration under the hood we need to able to flexibly handle an arbitrary number reference sections and associated reference sensors. Any changes on this will likely kill backwards compatibility.

My proposal is to merge the location library for the calibration points, data properties where we define probe names, into the calibration section. I'm picturing something like this:

```
calibration:
  builtin_probe_names:
    probe1Temperature: <reference sensor name>
    probe2Temperature: <other reference sensor name>
  
  library:
    <reference section short name>:
      long name: <reference section long name>
      LAF:
        core A:
        - LAF_A1
        - LAF_A2
        core B:
        - LAF_B1
        - LAF_B2
      ....
      ref_sensor: <reference sensor name>
      type: <validation or calibration> # Here we can specify a bath to withhold for validation
```
This would allow us to define any number of baths for validation or calibration at a later point and would put the critical reference section information in a single location.

**Comment by klapo** [comment]
Prior to calibration need to add renaming the built-in reference probes to be consistent with the config file. This fix would address changing names between iterations of data checks and calibrations and enable processing the data in independent chunks.

**Comment by klapo** [comment]
Closed by #39
